src/Swarm/Language/Syntax.hs

```diff
@@ -489,6 +492,7 @@ constInfo c = case c of
   Format -> functionLow 1
   Concat -> binaryOp "++" 6 R
   AppF -> binaryOp "$" 0 R
+  Teleport -> commandLow 2
```

Comment:
Shouldn't this be `commandLow 1`?  It only takes one argument.

Reply:
It should, I am surprised that it worked. :open_mouth: